ecommerce_extensions/settings/production.py

```diff
@@ -3,3 +3,28 @@
 """
 
 from __future__ import unicode_literals
+
+try:
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
+except ImportError:
+    sentry_sdk = DjangoIntegration = None
+
+# Sentry Integration
+    sentry_integration_dsn = "https://6b0a5c7f98b14a8bb3998aa4345ae9c4@o94678.ingest.sentry.io/5749202"
```

Comment:
This should not be get from an environment variable?

Reply:
I move that to get it from secure data

Reply:
Please make sere that this DSN was removed from the sentry platform and we generated a new one. @NeOneSoft